sessions/react-styled-components/lotr-app-styling/README.md

```diff
@@ -0,0 +1,104 @@
+# Styled Components: Lord of the Rings
+
+In this challenge you will finally style the LotR-Fanpage with the help of `styled-components`!
+
+> 💡 You can use this template as a starting point. But if you are far enough along with your own LotR-Fanpage App, please use that instead.
+
+## Task
+
+Use `styled-components` to create the design you can find in the [design files for the LotR-Fanpage](./README.md#resources).
+
+Before you start, follow the instructions to add all necessary files to your project.
+
+### Details Page
+
+You might start with the details page. Note that the background colors for each volume are given in the [`data.js`](./lib/data.js) file.
+
+How could one style a component depending on a prop again?
+
+#### Google Font
+
+There are font variables already prepared in the [styles.css file](./README.md#resources), but the design suggest to use the Lora font.
+
+> 💡 `@next/font` is already preinstalled in this template.
+
+- Import the Lora font into your `styles.js`.
+- Create a variable to define the `subsets` and `style` key from Lora, like so:
+
+```js
+const lora = Lora({
+  subsets: ["latin"],
+  style: ["normal", "italic"],
+});
+```
+
+- Finally, expand the `--font-family` variable to `${lora.style.fontFamily}, serif;`
+
+You should now be able to use all font variables! 🎉
+
+> 💡 If you get an error message, read it carefully: you might need to run `npm run dev` again and **save the styles file again** (without actually changing anything).
+
+#### SVG
+
+The [design files](./README.md#resources) also provide three svg files with arrows which you should have placed in an `icons` folder by now.
+
+In order to use the SVGs as React components, you need to install the [npm package SVGR](https://react-svgr.com/docs/next/) first. Follow the instructions given in the docs.
+
+To be sure how to adapt the `next.config.js` file, simply add the `webpack` function without a key:
+
+```js
+const nextConfig = {
+  // {...}
+  images: {
+    domains: ["upload.wikimedia.org"],
+  },
+  webpack(config) {
+    config.module.rules.push({
+      test: /\.svg$/i,
+      issuer: /\.[jt]sx?$/,
+      use: ["@svgr/webpack"],
+    });
+    return config;
+  },
+};
+```
+
+### All Volumes Page
+
+To create the box shadow of the books, you can use the predefined variables.
```

Comment:
We should add a note that the random button is no longer required (for students that solved the bonus challenge)